integr8ly/integreatly-operator
# Monitoring resources

Update monitoring to v1.0.0

Verification steps:

* Install monitoring using this branch
* Make sure that all operators, prometheus, alertmanager and grafana come up successfully
* Test with other products and make sure that all alerts, dashboards and blackboxtargets are imported
